### PR TITLE
Blog app - creating a data model

### PR DESCRIPTION
**Pull request by VaheStepanyan100**
- Be sured to reference the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) from the sneak peek if have any questions about table or column names or data relationships.
- Ensured that project has Postgres database set up.
- Builded project schema.
  - Create and run the necessary migration files.
  - Table and column names should match the ERD diagram.
      - note: photo for users table should be a link to an image
  - Foreign keys should be included.
    - Pay attention that in posts table the column holding the foreign key to the users table should be named author_id
  - All columns that are foreign keys should have a corresponding index.

### CODE REVIEW

**Review by TedLivist — COMMENTED** [review]
## STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @VaheStepanyan100,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### Kindly remove the live demo section and its link on the Table of Contents if you have no live demo link.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: db/schema.rb [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.0].define(version: 20_230_821_115_739) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension 'plpgsql'
+
+  create_table 'comments', force: :cascade do |t|
+    t.bigint 'author_id_id'
+    t.bigint 'post_id_id'
```
Comment (TedLivist) [inline db/schema.rb]: - [ ] These should be `author_id` and `post_id`. Kindly fix your column table names by running a new migration and renaming the columns.

File: db/schema.rb [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    t.text 'text'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.index ['author_id_id'], name: 'index_comments_on_author_id_id'
+    t.index ['post_id_id'], name: 'index_comments_on_post_id_id'
+  end
+
+  create_table 'likes', force: :cascade do |t|
+    t.bigint 'author_id_id'
+    t.bigint 'post_id_id'
```
Comment (TedLivist) [inline db/schema.rb]: - [ ] These should be `author_id` and `post_id`. Kindly fix your column table names by running a new migration and renaming the columns.

File: db/schema.rb [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.index ['author_id_id'], name: 'index_likes_on_author_id_id'
+    t.index ['post_id_id'], name: 'index_likes_on_post_id_id'
+  end
+
+  create_table 'posts', force: :cascade do |t|
+    t.bigint 'author_id_id'
```
Comment (TedLivist) [inline db/schema.rb]: - [ ] Kindly rename your table column to `author_id`

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @VaheStepanyan100,

Good job so far!
✔️ Use of PostgreSQL for database
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi @VaheStepanyan100 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
 Changes requested have been implemented ✅ 
   

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**